docs/UserGuide.md

```diff
@@ -195,10 +218,19 @@ Format: `unmark ANIMALINDEX TASKINDEX [TASKINDEX]...`
 * The `TASKINDEX` refers to the index of the task on the task list of the animal.
 * If multiple `TASKINDEX` are specified, all the tasks at the specified `TASKINDEX` will be marked as uncompleted.
 * `TASKINDEX` must be separated by a space.
+* If one of the `TASKINDEX` provided is invalid, the rest of the `TASKINDEX` will not be marked as uncompleted.
+* An error would not be thrown if the task(s) was previously marked as uncompleted or was still uncompleted, it would remain as uncompleted.
```

Comment:
Same here!

I think a better way to phrase it would be "Executing the `unmark` command on task(s) previously marked as uncomplete will not cause an error to be thrown, and the task(s) would remain as uncomplete (unmarked)."

Something along those lines, feel free to tweak it as you deem fit!